packages/docs/page-config/ui-elements/layout/examples/Order.vue

```diff
@@ -0,0 +1,102 @@
+<script setup lang="ts">
```

Comment:
Also would be nice to have a plausible mobile example.

<img width="680" alt="image" src="https://github.com/epicmaxco/vuestic-ui/assets/5394573/21173867-3835-47ca-846a-d7df177c50df">

Maybe we can just make sidebars proportional on mobile?